doc/man3/SSL_CTX_set_num_tickets.pod

```diff
@@ -0,0 +1,58 @@
+=pod
+
+=head1 NAME
+
+SSL_set_num_tickets,
+SSL_get_num_tickets,
+SSL_CTX_set_num_tickets,
+SSL_CTX_get_num_tickets
+- control the number of TLSv1.3 session tickets that are issued
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_set_num_tickets(SSL *s, size_t num_tickets);
+ size_t SSL_get_num_tickets(SSL *s);
+ int SSL_CTX_set_num_tickets(SSL_CTX *ctx, size_t num_tickets);
+ size_t SSL_CTX_get_num_tickets(SSL_CTX *ctx);
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_num_tickets() and SSL_set_num_tickets() can be called for a server
+application and set the number of session tickets that will be sent to the
+client after a full handshake. Set the desired value in the B<num_tickets>
+argument. Typically these functions should be called before the start of the
+handshake. They may be called later to control the number of tickets that will
+be issued on receipt of a post-handshake certificate from the client (after an
+initial full handshake). The default number of tickets is 2. The default number
+of tickets sent following a resumption handshake is 1. This cannot be changed
```

Comment:
"This cannot" meaning tickets after resumption?  Join the sentences with a semicolon and join the third using "but"  That's a long sentence but too many pronouns are confusing.  Maybe start a new paragraph before this sentence.

Reply:
Done